ivre/tools/scancli.py

```diff
@@ -476,6 +481,17 @@ def displayhost_csv(fields, separator, nastr, dic, out=sys.stdout):
                         for line in utils.doc2csv(dic, fields, nastr=nastr)))
     out.write('\n')
 
+def analyzer_init_context(hostfilter, skip=None, limit=None):
```

Comment:
For the sake of clarity, could you define this function only when `USING_CONTEXT`?